UnifiedMap/UnifiedMap.Droid/UnifiedMapRenderer.cs

```diff
@@ -465,6 +531,9 @@ private async Task AddPinAsync(IMapPin pin)
 
             mapPin.InvokeZIndex(pin.ZIndex);
 
+            // For draggable and LongPress Events
+            mapPin.Draggable(true);
```

Comment:
What if the User of UnifiedMaps wants to disable Dragging? Would it be better if there is a property like `DraggablePinsEnabled` which can be enabled or disabled?

Reply:
The Draggable property is a property of Unified IMapPin.
You can enable it or disabled it. 
The Fivenine Pin is set to draggable true on the sample, the others not.

On Android to get a LongPressGesture on a pin, I had to enable Draggable property and stop the event at star. It become a longPress.
That why the longPress gesture bring me to the drags gestures.

And Drag gesture set to true is disabling LongPress. Can't have both.

Reply:
funny, it did not appear in the Diff view in "Files changed"